src/components/volunteer-enroll/VolunteerEnrollFormWrapper.tsx

```diff
@@ -0,0 +1,137 @@
+                                                  )
+                                            : null
+                                    }
+                                />
+                                <CloseButton />
+                            </Flex>
+                            {formPages.map((formPage, idx) => {
+                                return (
+                                    <Box
+                                        key={idx}
+                                        display={
+                                            pageNum === idx ? null : "none"
+                                        }
+                                    >
+                                        {formPage}
+                                    </Box>
+                                );
+                            })}
+                        </Box>
+                    </Center>
+                ) : (
+                    <Center>
+                        <VStack mt={120} mb={180}>
+                            {/* need to add check icon? doesn't exist on other pages */}
+                            <Text
+                                mb={"55px"}
+                                maxW={476}
+                                fontWeight="700"
+                                fontSize="36px"
+                                align="center"
+                            >
+                                Thank you for signing up to volunteer!
+                            </Text>
+                            <Text maxW={512} textAlign="center">
+                                We're really excited that you want to volunteer
+                                for us. Look out for an email from us shortly
+                                with more information!
+                            </Text>
+                            <Center>
+                                <Link
+                                    _hover={{ textDecoration: "none" }}
+                                    _focus={{}}
+                                    href="/"
+                                >
+                                    <Button
+                                        mt={"55px"}
+                                        borderColor={colourTheme.colors.Blue}
+                                        border="2px"
+                                        width={"364px"}
+                                        height={"49px"}
+                                        color={colourTheme.colors.Blue}
+                                        backgroundColor={
+                                            colourTheme.colors.white
+                                        }
+                                        px={10}
+                                        _hover={{
+                                            bg: colourTheme.colors.LightBlue,
+                                        }}
```

Comment:
This hover colour don't look right lol
![image](https://user-images.githubusercontent.com/44826218/138012804-7fa2e216-19ae-4400-adf3-3e3ab86d87a7.png)
```suggestion
```